velox/type/DecimalUtils.cpp

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
can we move the DecimalUtils to another PR to keep this PR short?

Reply:
https://github.com/facebookincubator/velox/pull/2019/